modules/bim/game/main/src/bim/game/system/update_bomb_power_up_spawners.cpp

```diff
@@ -2,10 +2,12 @@
 #include <bim/game/system/update_bomb_power_up_spawners.hpp>
 
 #include <bim/game/component/bomb.hpp>
+#include <bim/game/component/bomb_power_up.hpp>
```

Comment:
Interestingly we can avoid the include directive here, a simple forward declaration is enough! Just add the following code after all include directives:

```c++
namespace bim::game
{
  struct bomb_power_up;
}
```
There are more occurrences of this that I don't tag in the other files.